openmc/cell.py

```diff
@@ -139,7 +143,10 @@ def __repr__(self):
             string += '\t{0: <15}=\t{1}\n'.format('Temperature',
                                                   self.temperature)
         string += '{: <16}=\t{}\n'.format('\tTranslation', self.translation)
-        string += '{: <16}=\t{}\n'.format('\tVolume', self.volume)
+        if self._volume_std is not None:
+            string += '{: <16}=\t{} +/- {}\n'.format('\tVolume', self.volume, self.volume_std)
+        else:
+            string += '{: <16}=\t{}\n'.format('\tVolume', self.volume)
```

Comment:
```suggestion
            string += '{: <16}=\t{} +/- {} [cm]\n'.format('\tVolume', self.volume, self.volume_std)
        else:
            string += '{: <16}=\t{} [cm]\n'.format('\tVolume', self.volume)
```

Would it be ok to add in units here, I notice we have units for other repr entries such as density

Reply:
yes, good idea, though it should be `[cm^3]`

Reply:
Yes good spot, I have updated the suggestion above to cm^3